models/test/mapper/TestPreviewMapper.php

```diff
@@ -134,7 +134,7 @@ public function map(AssessmentTest $test, Route $route, TestPreviewConfig $confi
 
                 if (!isset($map['parts'][$partId]['sections'][$sectionId])) {
                     $map['parts'][$partId]['sections'][$sectionId]['id'] = $sectionId;
-                    $map['parts'][$partId]['sections'][$sectionId]['label'] = html_entity_decode($section->getTitle());
+                    $map['parts'][$partId]['sections'][$sectionId]['label'] = $section->getTitle();
```

Comment:
Can you please just confirm that tests are still passing? Thanks!

Reply:
@gabrielfs7 no change in the result of the unit tests